zeppelin-web/Gruntfile.js

```diff
@@ -55,15 +55,15 @@ module.exports = function(grunt) {
           expand: true,
           cwd: './src/',
           src: ['**/*.js'],
-          dest: '.tmp',
```

Comment:
You don't have to change it everywhere, but you can if you want to. Or you could just leave things as they were before. Up to you.